shared/teams/team/new-header.tsx

```diff
@@ -316,9 +316,7 @@ const useHeaderCallbacks = (teamID: TeamID) => {
     ? () =>
         dispatch(
           nav.safeNavigateAppendPayload({
-            path: [
-              {props: {sendChatNotification: true, teamname: meta.teamname}, selected: 'teamEditTeamAvatar'},
-            ],
+            path: [{props: {sendChatNotification: true, teamID}, selected: 'profileEditAvatar'}],
```

Comment:
There isn't a way to upload an avatar still with this. We should have the same flow as with the flag off.

Reply:
Fixed by adding to this wrapper. I'd rather we move away from manually launching that sheet before navigating to this modal, it's kinda confusing.

Reply:
sgtm, no strong opinion.